src/chain_spec.rs

```diff
@@ -79,6 +79,57 @@ pub fn charcoal_local_network(para_id: ParaId) -> ChainSpec {
 	)
 }
 
+pub fn altair_staging_network(para_id: ParaId) -> ChainSpec {
+	let mut properties = Properties::new();
+	properties.insert("tokenSymbol".into(), "AIR".into());
+	properties.insert("tokenDecimals".into(), 18.into());
+
+	ChainSpec::from_genesis(
+		"Altair",
+		"altair",
+		ChainType::Live,
+		move || {
+			testnet_genesis(
+				hex!["66d97d3816f5906c8a9821fac25afbb76291b12eb51c5a559e44aaafe4e42206"].into(),
+				vec![
+					(
+						//
+						hex!["b24fb587438bbe05034606dac98162d80be1d21ac6dd6edc989887fa53a8d503"]
+							.into(),
+						hex!["c475e1ba26aa503601f26568ce6989502fc316b41c6d788b58e4cba4ec967a73"]
+							.unchecked_into(),
+					),
+					(
+						//
+						hex!["d46783c911c4d8fb42f8239eb8925857e27ee3bdd121feb43e450241891a5f1e"]
+							.into(),
+						hex!["4ab9526ff43c29426a6288621d85e3cbd45bcb279eab1cf250079b02d2a40e2f"]
+							.unchecked_into(),
+					),
+					(
+						//
+						hex!["f02099f295f6ccd935646f50c6280f4054b7d1f9b126471668f4ac6175677c26"]
+							.into(),
+						hex!["2652d9800f7dcca7592c83857ecc674f34a51f7661d6dc06281565557e5ee217"]
+							.unchecked_into(),
+					),
+				],
+				vec![],
+				None,
```

Comment:
@vedhavyas Removed total issuance, since we are going to set it up as part of the migration, is it ok?